modules/ztp-image-based-upgrade-installing-lcao-with-gitops.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-installing-lcao-with-gitops_{context}"]
+= Installing the {lcao} with GitOps ZTP
+
+Install the {lcao} with GitOps ZTP to do an image-based upgrade.
+
+.Prerequisites
+
+* Create a directory called `custom-crs` in the `source-crs` directory. The `source-crs` directory must be in the same location as the `kustomization.yaml` file.
+
+.Procedure
+
+. Create the following CRs in the `openshift-lifecycle-agent` namespace and push them to the `source-crs/custom-crs` directory:
++
+--
+.LcaSubscriptionNS.yaml
```

Comment:
Throughout the PR, for any of the code blocks where the title is just the file name, I'd suggest marking them up and adding the following:
```suggestion
.Example `LcaSubscriptionNS.yaml` file
```
or if you don't want "Example" (if you want it clear that you should explicitly use this configuration), then at least:
```suggestion
.`LcaSubscriptionNS.yaml` file
```